include/libtorrent/alert_types.hpp

```diff
@@ -2064,6 +2065,12 @@ namespace libtorrent
 		boost::array<char, 64> signature;
 		std::string salt;
 		boost::uint64_t seq;
+
+		// DHT put operation usually writes item to k nodes, maybe the node
+		// is stale so no response, or the node doesn't support 'put', or the
+		// token for write is out of date, etc.
+		// set success to true when success responses >= failed responses.
```

Comment:
yes. 

Good idea, actually I remember I wrote similar idea (the number of confirmed successful and also the number of time out) here but I can't find it.